### ISSUE TITLE
reSTLintBearTest fails on windows and mac os x because of locales

### ISSUE THREAD

**Issue by sudheesh001** [coala-analyzer/coala#1296]
An easy way to fix this would be to integrate
```py
import locale
locale.setlocale()
```
This can be open to someone to take and fix as their first bug

**Comment by sils1297** [comment]
Integrate it where? Is this related to your fix in setup.py?

**Comment by sudheesh001** [comment]
No, this is unrelated to that, that was to set coala up on the system, this is needed to run the tests, specifically this one

**Comment by sils1297** [comment]
@sudheesh001 for this to be newcomer suitable please provide a more specific description

**Comment by sudheesh001** [comment]
The test reSTLintBearTest fails on mac os x and windows because the test raises an error
```
    raise ValueError('unknown locale: %s' % localename)
ValueError: unknown locale: UTF-8
```
This can be fixed by setting the locale to UTF8 before the tests start to run.
